lib/passport-saml/config/schema.js

```diff
@@ -1,8 +1,8 @@
-const SCHEME = {
+var SCHEME = {
```

Comment:
It would be helpful if the commit message explained why it was necessary to replace `const` with `var` here.

Reply:
With node ver4.0, Travis fails to pass. In reality:
`'const' Declarations in non-strict mode are not supported yet on Node v4.`

For the sake of consistency (like the other files), I prevented the use of strict mode.

Reply:
Thanks for the great points! 👍 

I'd rather create a migration branch, to have things more transparent. Can we continue with the fix as it is, and consider that migration as a separate task?